Cargo.toml

```diff
@@ -74,8 +72,8 @@ futures = "0.3.26"
 futures-util = "0.3"
 dashmap = "5.4.0"
 
-relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", tag = "v0.26.2", features = ["cacao"] }
-relay_client = { git = "https://github.com/WalletConnect/WalletConnectRust.git", tag = "v0.26.2" }
+relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", branch = "53-upgrade-ed25519-dalek-dependency-to-200", features = ["cacao"] }
```

Comment:
will the contents of these branches be merged into main anytime soon?

Reply:
Yes see PR description

Reply:
Using branch so it passes CI and will update to tagged version before merging